source/about/accessibility_html_error_messages.md

```diff
@@ -0,0 +1,54 @@
+# Error messages in HTML papers
+
+HTML papers on arXiv.org are a work in progress and will sometimes display errors. As we work to improve accessibility we share with you the causes of these errors and what authors can do to help minimize them.
+{ .intro }
+
+To produce accessible HTML papers we use a converter created and maintained by the LaTeXML team at the National Library of Standards and Technology (NIST). 
+
+The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software constructs. TeX and LaTeX, pronounced ‘tech’ and ‘LAH-tech’, are open source software packages used to typeset science and math papers. TeX and LaTeX made it possible accurately display complex mathematical formulae, symbols and tables in research papers. TeX is a fully programmable layout system, and there are many ways to extend the capabilities of TeX by creating packages of TeX code and macros. In fact, LaTeX itself is simply a large package of TeX extensions. The LaTeX community uses a large number of freely-available packages that are used by the scientific community for tasks like adding figures to articles, or following the publication template for a journal. All of the packages have to be individually supported in LaTeXML. While LaTeXML supports nearly all of the most commonly used LaTeX packages, there is a long tail of less commonly used packages that are not presently supported. The LaTeXML team will continue adding packages, but it will take a very long time to support every package used by every paper on arXiv.
```

Comment:
Almost all of this paragraph is repeated in "What is TeX" and "What is LaTeX" below. Maybe leave jus:

> The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software constructs

And merge the rest of the paragraph into the "What is ... " section below. Particular since I suspect 95% of arXiv readers already know what TeX and LaTeX users are.

Perhaps one question: "What is TeX and LaTeX"? followed by "Why do many of the HTML papers have conversion artifacts" which would have the discussion about unsupported packages?